siteorigin-panels.php

```diff
@@ -732,11 +732,14 @@ function siteorigin_panels_generate_css($post_id, $panels_data = false){
 			));
 		}
 
+
+		$collapse_order = !empty( $grid['style']['collapse_order'] ) ? $grid['style']['collapse_order'] : ( !is_rtl() ? 'left-top' : 'right-top' );
```

Comment:
Ok, I think it's fair enough either way. Just wondered about the case where a user might want the opposite of the deault `is_rtl()` behaviour.